admin-frontend/src/services/apiService.ts

```diff
@@ -340,17 +340,24 @@ export default {
   },
 
   async addAnnouncement(data: AnnouncementFormValue) {
+    const formData = buildFormData(data);
+
     try {
-      return await apiAxios.post(ApiRoutes.ANNOUNCEMENTS, data);
+      return await apiAxios.post(ApiRoutes.ANNOUNCEMENTS, formData, {
+        headers: { 'Content-Type': 'multipart/form-data' },
+      });
     } catch (e) {
       console.log(`Failed to post from Nodejs addAnnouncement API - ${e}`);
```

Comment:
Update all the console.log and error

Reply:
Do you mean change from console.log to console.error? @sukanya-rath